diff --git a/internal/cmd/main.go b/internal/cmd/main.go
index 88184699..3d039279 100644
--- a/internal/cmd/main.go
+++ b/internal/cmd/main.go
@@ -171,7 +171,7 @@ func Main(ctx context.Context, args []string, stdin io.Reader, stdout, stderr io
 	command.SetArgs(args)
 	command.SetIn(stdin)
 	command.SetOut(stdout)
-	command.SetErr(stderr)
+	// command.SetErr(stderr)
 
 	err := command.ExecuteContext(ctx)
 	if err != nil {
